src/test/java/org/candlepin/util/UtilTest.java

```diff
@@ -601,4 +604,42 @@ void name() {
         assertEquals(true, Util.isFalse(false));
         assertEquals(false, Util.isFalse(true));
     }
+
+    @Test
+    void testHashedPasswordsProduceHash() {
+        String source = "source_string";
+
+        String hash = Util.hashPassword(source);
+
+        assertNotNull(hash);
+    }
+    @Test
+    void testHashedPasswordsShouldNotEqualForSameInput() {
+        String source = "source_string";
+
+        String hash1 = Util.hashPassword(source);
+        String hash2 = Util.hashPassword(source);
+        String hash3 = Util.hashPassword(source);
+
+        assertNotEquals(hash1, hash2);
```

Comment:
As mentioned above, this is not a good way to go about this. If we cannot guarantee validity and consistency, the function's behavior is effectively undefined. We've gone from hash function (fixed input = fixed, but different, output) to random password generation tied to a specific library/algorithm.

The function name still has "hash" in it, which implies something very different than what we're actually doing now, but with no documentation or anything to actually outline what it's doing.